mammoth/docx/body_xml.py

```diff
@@ -265,8 +266,11 @@ def bookmark_start(element):
     def br(element):
         break_type = element.attributes.get("w:type")
         if break_type:
-            warning = results.warning("Unsupported break type: {0}".format(break_type))
-            return _empty_result_with_message(warning)
+            if break_type == 'page':
+                return _success(documents.line_break("pageBreak"))
```

Comment:
I think the `Break` suffix feels redundant for the break type. Perhaps just `documents.break("page")`?